openvdb/openvdb/tree/InternalNode.h

```diff
@@ -274,9 +274,18 @@ class InternalNode
     /// Set the transient data value.
     void setTransientData(Index32 transientData) { mTransientData = transientData; }
 
+#if OPENVDB_ABI_VERSION_NUMBER >= 12
+    Index64 leafCount() const;
+    Index64 nonLeafCount() const;
+#else
     Index32 leafCount() const;
-    void nodeCount(std::vector<Index32> &vec) const;
     Index32 nonLeafCount() const;
+#endif
+    void nodeCount(std::vector<Index64> &vec) const;
+#if OPENVDB_ABI_VERSION_NUMBER < 12
```

Comment:
Ideally, we should be aiming to offer both `nodeCount(std::vector<Index32>&)` and nodeCount(std::vector<Index64>&)` for this release except that the 32-bit one is marked as deprecated. If we make the 32-bit one disappear in 12.0 ABI=12, this will be a breaking API change without a chance for people to see the deprecation message.